cmake/CMakeLists.txt

```diff
@@ -707,9 +707,9 @@ endif()
 
 if (onnxruntime_BUILD_BENCHMARKS)
   if (onnxruntime_PREFER_SYSTEM_LIB)
-    find_package(benchmark)
+    find_package(benchmark CONFIG)
   endif()
-  if (benchmark_FOUND)
+  if (TARGET benchmark::benchmark)
```

Comment:
Should we use `benchmark::benchmark` in some `target_link_librarires`?

Reply:
I only found `benchmark::benchmark` for `target_link_libraries`:
- https://github.com/microsoft/onnxruntime/blob/master/cmake/onnxruntime_unittests.cmake#L895
- https://github.com/microsoft/onnxruntime/blob/master/cmake/onnxruntime_unittests.cmake#L903

Did I miss something?

Reply:
Ah yes, sorry for disturb.